### PR TITLE
Annotate multiline TODO comments

### PR DESCRIPTION
**Pull request by prestonvasquez**
[GODRIVER-2397](https://jira.mongodb.org/browse/GODRIVER-2397)

### CODE REVIEW

File: bson/bsonrw/value_reader.go [hunk 1]
```diff
@@ -87,10 +87,10 @@ type valueReader struct {
...
 //
-// TODO(skriptble): There's a lack of symmetry between the reader and writer, since the reader takes
-// a []byte while the writer takes an io.Writer. We should have two versions of each, one that takes
-// a []byte and one that takes an io.Reader or io.Writer. The []byte version will need to return a
-// thing that can return the finished []byte since it might be reallocated when appended to.
+// TODO(skriptble): There's a lack of symmetry between the reader and writer, since the reader takes a []byte while the
+// TODO writer takes an io.Writer. We should have two versions of each, one that takes a []byte and one that takes an
+// TODO io.Reader or io.Writer. The []byte version will need to return a thing that can return the finished []byte since
+// TODO it might be reallocated when appended to.
 func NewBSONDocumentReader(b []byte) ValueReader {
```
Comment (matthewdale) [inline bson/bsonrw/value_reader.go:94]: We should keep TODO comments out of the exported API documentation. Move the TODO comment block to the first line of the `NewBSONDocumentReader` function.

File: mongo/integration/mtest/mongotest.go [hunk 2]
```diff
@@ -268,8 +268,9 @@ func (t *T) ClearMockResponses() {
 
 // GetStartedEvent returns the most recent CommandStartedEvent, or nil if one is not present.
 // This can only be called once per event.
-// TODO(GODRIVER-2075): GetStartedEvent documents that it returns the most recent event, but
-// actually returns the first event. Update either the documentation or implementation.
+//
+// TODO(GODRIVER-2075): GetStartedEvent documents that it returns the most recent event, but actually returns the first
+// TODO event. Update either the documentation or implementation.
 func (t *T) GetStartedEvent() *event.CommandStartedEvent {
```
Comment (matthewdale) [inline mongo/integration/mtest/mongotest.go:274]: Same as above, move the TODO comment block to the first line of the `GetStartedEvent` function.

File: mongo/integration/mtest/mongotest.go [hunk 3]
```diff
@@ -281,8 +282,9 @@ func (t *T) GetStartedEvent() *event.CommandStartedEvent {
 
 // GetSucceededEvent returns the most recent CommandSucceededEvent, or nil if one is not present.
 // This can only be called once per event.
-// TODO(GODRIVER-2075): GetSucceededEvent documents that it returns the most recent event, but
-// actually returns the first event. Update either the documentation or implementation.
+//
+// TODO(GODRIVER-2075): GetSucceededEvent documents that it returns the most recent event, but actually returns the
+// TODO first event. Update either the documentation or implementation.
 func (t *T) GetSucceededEvent() *event.CommandSucceededEvent {
```
Comment (matthewdale) [inline mongo/integration/mtest/mongotest.go:288]: Same as above, move the TODO comment block to the first line of the `GetSucceededEvent` function.

File: mongo/integration/mtest/mongotest.go [hunk 4]
```diff
@@ -294,8 +296,9 @@ func (t *T) GetSucceededEvent() *event.CommandSucceededEvent {
 
 // GetFailedEvent returns the most recent CommandFailedEvent, or nil if one is not present.
 // This can only be called once per event.
-// TODO(GODRIVER-2075): GetFailedEvent documents that it returns the most recent event, but actually
-// returns the first event. Update either the documentation or implementation.
+//
+// TODO(GODRIVER-2075): GetFailedEvent documents that it returns the most recent event, but actually  returns the first
+// TODO event. Update either the documentation or implementation.
 func (t *T) GetFailedEvent() *event.CommandFailedEvent {
```
Comment (matthewdale) [inline mongo/integration/mtest/mongotest.go:302]: Same as above, move the TODO comment block to the first line of the `GetFailedEvent` function.

File: x/bsonx/bsoncore/bsoncore.go [hunk 5]
```diff
@@ -198,10 +197,10 @@ func ReadString(src []byte) (string, []byte, bool) {
...
 //
-// TODO(skriptble): We really need AppendDocumentStart and AppendDocumentEnd.
-// AppendDocumentStart would handle calling ReserveLength and providing the index of the start of
-// the document. AppendDocumentEnd would handle taking that start index, adding the null byte,
-// calculating the length, and filling in the length at the start of the document.
+// TODO(skriptble): We really need AppendDocumentStart and AppendDocumentEnd.  AppendDocumentStart would handle calling
+// TODO ReserveLength and providing the index of the start of the document. AppendDocumentEnd would handle taking that
+// TODO start index, adding the null byte, calculating the length, and filling in the length at the start of the
+// TODO document.
 func AppendDocumentStart(dst []byte) (index int32, b []byte) { return ReserveLength(dst) }
```
Comment (matthewdale) [inline x/bsonx/bsoncore/bsoncore.go:204]: Same as above, move the TODO comment block to the first line of the `AppendDocumentStart` function.

**Review by benjirewis — APPROVED** [review]
LGTM mod Matt's comments about moving `TODO`s out of documentation comments.

**Review by matthewdale — APPROVED** [review]
Looks good 👍

**Pull request merged**